### PR TITLE
OCPQE-14455: e2e on ROSA PROD hostedcluster

### PR DESCRIPTION
**Pull request by LiangquanLi930**

### CODE REVIEW

**Comment by LiangquanLi930** [comment]
@liangxia Could you help to review this PR? thanks

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-stable.yaml [hunk 1]
```diff
@@ -394,6 +394,25 @@ tests:
     - ref: cucushift-helper-service-build-info
     - ref: openshift-extended-test
     workflow: rosa-aws-sts-hypershift
+- as: aws-rosa-sts-hypershift-prod-guest-p1-f7
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-stable.yaml:397]: @YuLi517 @jhou1 Will hypershift need to follow the test strategy for nightly/stable payloads we did recently ?

Comment (YuLi517) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-stable.yaml:397]: This is for rosa-hypershift. rosa env can only use stable image to run the test.

Comment (liangxia) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-stable.yaml:397]: Ack

**Comment by liangxia** [comment]
This branch has conflicts that must be resolved

**Comment by LiangquanLi930** [comment]
@liangxia Thanks, Could you help to approve it?

**Pull request merged**